docs/getting-started/concepts/feature-view.md

```diff
@@ -53,6 +53,12 @@ global_stats_fv = FeatureView(
 {% endtab %}
 {% endtabs %}
 
+## Feature inferencing
+
+If the `features` param is not specified in the FeatureView creation, Feast will infer the features during `feast apply` by creating a feature for each column in the
+underlying data source except the columns corresponding to the entities of the FeatureView or the columns corresponding to the timestamp columns of the FeatureView's
```

Comment:
Nit: we prefer using the term 'feature view' instead of 'FeatureView'. Same for the other page.